### PR TITLE
added whereisclown to check last tweet from forsen

### PR DESCRIPTION
**Pull request by sakurazaki**

### CODE REVIEW

File: internal/commands/whereisstreamer/whereisstreamer.go [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+	"github.com/bwmarrin/discordgo"
+	"github.com/pajbot/basecommand"
+	"github.com/pajbot/pajbot2-discord/internal/config"
+	"github.com/pajbot/pajbot2-discord/pkg"
+	"github.com/pajbot/pajbot2-discord/pkg/commands"
+)
+
+func init() {
+	commands.Register([]string{"$whereisclown"}, New())
```
Comment (sakurazaki) [inline internal/commands/whereisstreamer/whereisstreamer.go]: added whereisclown for forsen despite being whereisstreamer. Maybe we want to make it generic or make it configurable?

File: internal/commands/whereisstreamer/whereisstreamer.go [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+		log.Println(err)
+		return
+	}
+	
+	if len(tweets) < 1 {
+		s.ChannelMessageSend(m.ChannelID, "There was an error requesting the tweet information.")
+		return
+	}
+
+	const resultFormat = "Last tweet from clown: https://twitter.com/%s/status/%s"
```
Comment (sakurazaki) [inline internal/commands/whereisstreamer/whereisstreamer.go]: Same as above I added clown instead of forsen or streamer name.

**Comment by pajlada** [comment]
Twitter auth can still be stored in the environment files, that's fine.
The username should be stored in the database with the "serverconfig" stuff.

**Comment by sakurazaki** [comment]
Moving to a new pull request

**Pull request closed without merging**